fedbiomed/common/training_plans/_base_training_plan.py

```diff
@@ -81,6 +81,17 @@ def __init__(self) -> None:
     def model(self):
         """Gets model instance of the training plan"""
 
+    # FIXME: re-implement Model as a subclass of Torch.nn.module, SGDClassifier, etc.
+    # to avoid having a distinct getter for the class, see #1049
+
+    def get_model_wrapper_class(self) -> Model:
+        """Gets training plan's model wrapper class.
```

Comment:
Thank you @ybouilla 
You are right, thank you for fixing the typing.
So I also updated typing for `self._model` in `BaseTrainingPlan` (it was `Model` instead of `Optional[Model]`, which is not coherent avec sub classes)

Would you mark the review as accepted so I merge if OK for you ?